scripts/run-benchmarks.main.kts

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env kotlin
+
+@file:DependsOn("com.squareup.okio:okio:3.2.0")
+@file:DependsOn("com.google.cloud:google-cloud-storage:2.8.1")
+@file:DependsOn("net.mbonnin.bare-graphql:bare-graphql:0.0.2")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.3")
+
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageOptions
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import net.mbonnin.bare.graphql.asList
+import net.mbonnin.bare.graphql.asMap
+import net.mbonnin.bare.graphql.cast
+import net.mbonnin.bare.graphql.graphQL
+import okio.Buffer
+import okio.buffer
+import okio.source
+import java.io.File
+import java.util.Date
+
+
+/**
+ * This script expects:
+ *
+ * - `gcloud` in the path
```

Comment:
Is gcloud present by default on the GH actions image, or did I miss something?

Reply:
Good question. Looks like [it is preinstalled indeed](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#cli-tools)

Reply:
Handy! 👍